.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -61,12 +61,10 @@ body:
       label: 📟 Stack Trace
       description: Copy and paste the stack trace from FluentHub's error page.
       value: |
-        "details><summary>Stack trace details</summary><div>
-        
+        <details><summary>Stack trace details</summary><div>
         ```
-        paste here
+        <!--paste here-->
```

Comment:
In code block, it doesn't make any sence.

Reply:
ohhh I forgot that..